terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/WebDriverOperations.java

```diff
@@ -44,6 +48,15 @@ public void setDefaultTimeoutForImplicitlyWait(
     }
 
     /**
+     * Get the text (display value) set for the specified element.
+     * @param by Identifier to look for elements
+     * @return And returns the text (display value)
+     */
+    public String getText(By by) {
+    	return webDriver.findElement(by).getText();
+    }

```

Comment:
Treat in https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/608#discussion_r104845354 too.